CMakeLists.txt

```diff
@@ -315,6 +315,31 @@ if(LAPACKE)
   add_subdirectory(LAPACKE)
 endif()
 
+# --------------------------------------------------
+# BLAS++ / LAPACK++
+option(BLAS++ "Build BLAS++" OFF)
+option(LAPACK++ "Build LAPACK++" OFF)
```

Comment:
To be consistent with the name of the projects, I suggest to use `BLASPP` and `LAPACKPP`

* https://bitbucket.org/icl/blaspp
* https://bitbucket.org/icl/lapackpp

Reply:
The projects are named BLAS++ and LAPACK++. The "blaspp" and "lapackpp" names are just because + doesn't play well as a filename. Bitbucket doesn't seem to allow + in a repo name.

Reply:
I still think the option should be consistent with the name of the cmake project and associated targets, see https://bitbucket.org/icl/blaspp/src/da11db2a6f85b402591005b05f233bda19aa4afe/CMakeLists.txt?at=default&fileviewer=file-view-default#CMakeLists.txt-7